contrib/platform/src/com/sun/jna/platform/win32/Psapi.java

```diff
@@ -0,0 +1,54 @@
+/* Copyright (c) 2015 Andreas "PAX" L\u00FCck, All Rights Reserved
+ * 
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * 
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.  
+ */
+package com.sun.jna.platform.win32;
+
+import com.sun.jna.Native;
+import com.sun.jna.platform.win32.WinNT.HANDLE;
+import com.sun.jna.win32.StdCallLibrary;
+
+/**
+ * The process status application programming interface (PSAPI) is a helper
+ * library that makes it easier for you to obtain information about processes
+ * and device drivers.
+ * 
+ * @author Andreas "PAX" L&uuml;ck, onkelpax-git[at]yahoo.de
+ */
+public interface Psapi extends StdCallLibrary {
+	public static final Psapi INSTANCE = (Psapi) Native.loadLibrary("psapi",
+			Psapi.class);

```

Comment:
This should be loaded with the same options as the other w32 libraries, namely with W32APIOptions.DEFAULT_OPTIONS.  That automatically handles `A/W` suffixes and wide string type mappings.